CefSharp.Wpf/ChromiumWebBrowser.cs

```diff
@@ -2442,33 +2501,30 @@ protected bool InternalIsBrowserInitialized()
             return Interlocked.CompareExchange(ref browserInitialized, 0, 0) == 1;
         }
 
-        private void ResizeHackFor2779()
+        private async Task ResizeHackFor2779()
         {
             if (EnableResizeHackForIssue2779)
             {
                 const int delayInMs = 50;
 
-                CefUiThreadRunAsync(async () =>
+                var host = browser?.GetHost();
+                if (host != null && !host.IsDisposed)
                 {
-                    if (browser != null)
-                    {
-                        resizeHackForIssue2779Size = new Structs.Size(viewRect.Width - 1, viewRect.Height - 1);
-                        browser.GetHost().WasResized();
-                    }
+                    resizeHackForIssue2779Size = new Structs.Size(viewRect.Width - 1, viewRect.Height - 1);
```

Comment:
I believe that this causes crashes on cef due when returning negative values. I was able to reproduce a crash because GetViewRect returns negetive width and height. Probably it would be safer to add 1px instead.